modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding a plugin to the {cli-manager}
+
+You can add or update a CLI plugin to the {cli-manager} using the YAML View.
+
+.Prerequisites
+
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* The {cli-manager} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
+
+. Click *Create Plugin*.
+
+. Enter the YAML file with the information for the plugin you are installing in the text box. See the example YAML file below.
++
+.Example YAML file to add a plugin
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Plugin
+metadata:
+  name: <plugin_name> <1>
+spec:
+  description: <description_of_plugin>
+  homepage: <plugin_homepage>
+  platforms:
+  - bin: <2>
+    files:
+    - from: <plugin_file_path>
+      to: .
+    image: <plugin_image>
+    imagePullSecret: <3>
+    platform: <platform> <4>
+  shortDescription: <short_description_of_plugin>
+  version: <version> <5>
+----
+<1> The name of the plugin you will use in commands.
+<2> `Bin` specifies the path to the plugin executable.
```

Comment:
Sounds reasonable and compatible with https://github.com/openshift/cli-manager/blob/dc93d208c9cd95606fdaaac45f93c652fef7c349/api/v1alpha1/plugin_types.go#L61-L66